scanner/config/config.go

```diff
@@ -171,6 +172,30 @@ type MatcherConfig struct {
 	VulnerabilityVersion string `yaml:"vulnerability_version"`
 }
 
+// resolveVersions returns values for ROX_VERSION and ROX_VULNERABILITY_VERSION
+// based on the current build information and version string name.  If the user
+// has explicitly set a VulnerabilityVersion in the configuration, it overrides
+// the default values to maintain backward compatibility with pre-existing
+// configs.
+func (c *MatcherConfig) resolveVersions() (roxVer, vulnVer string) {
+	roxVer = "dev"
+	vulnVer = "dev"
+	// Rely on buildinfo first to check release biulds, it is defined by
```

Comment:
```suggestion
	// Rely on buildinfo first to check release builds. It is defined by
```

Reply:
I don't have any other change requested for this, and this seems like a nit, so I will do it if any other change is needed.